Calculator.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+
+int main()
```

Comment:
Is it a good approach to write everything inside the main function? Divide the code into functions based on responsibility.